src/main/java/pl/nextleveldev/smart_route/infrastructure/um/UmWarsawResponseMapper.java

```diff
@@ -13,7 +15,7 @@ static UmBusLineResponse mapBusLineResponse(
                     response.result().stream()
                             .flatMap(resultValues -> resultValues.values().stream())
                             .filter(value -> "linia".equalsIgnoreCase(value.key()))
-                            .map(UmWarsawGenericResponse.Value::value)
+                            .map(UmWarsawGenericResponse.ResultValues.Value::value)
                             .toList();
```

Comment:
Im not sure but maybe it should be

resultValue instead of resultValue**s**

```suggestion
            List<String> lines =
                    response.result().stream()
                            .flatMap(resultValue -> resultValue.values().stream())
                            .filter(value -> "linia".equalsIgnoreCase(value.key()))
                            .map(UmWarsawGenericResponse.ResultValue.Value::value)
                            .toList();
```